bundles/AdminBundle/public/js/pimcore/object/classes/data/classificationstore.js

```diff
@@ -40,7 +40,7 @@ pimcore.object.classes.data.classificationstore = Class.create(pimcore.object.cl
     },
 
     getTypeName: function () {
-        return t("classificationstore");
+        return t("classification_store");
```

Comment:
why is this line changed?

Reply:
Because this translation is used in https://github.com/pimcore/pimcore/blob/0c15abf9691589c77fc488a76091ac163fe6d705/bundles/AdminBundle/public/js/pimcore/layout/toolbar.js#L626 and https://github.com/pimcore/pimcore/blob/0c15abf9691589c77fc488a76091ac163fe6d705/bundles/AdminBundle/public/js/pimcore/object/classificationstore/storeTree.js#L32

We do not need both imho.

Reply:
It seems that both versions exist ...

https://github.com/pimcore/pimcore/blob/c8f29107a6a39f29165bd909248b32a5a83e7ee6/bundles/AdminBundle/translations/admin_ext.en.yaml#L665

and 

https://github.com/pimcore/pimcore/blob/c8f29107a6a39f29165bd909248b32a5a83e7ee6/bundles/AdminBundle/translations/admin_ext.en.yaml#L732

I checked the occurences in the code and I would suggest going with `classificationstore` because this version is used more often. According to that, we could also delete the `classification_store` translations.

Reply:
WDYT?

Reply:
Ok, reverted to `classification_store`.
Changing the other translations which use `classification_store` should be done in separate PR.